satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -44,4 +44,29 @@ delete irreparabledb ( where irreparabledb.segmentpath = ? )
 read one (
   select irreparabledb 
   where  irreparabledb.segmentpath = ?
-)
\ No newline at end of file
+)
+
+// dbx.v1 golang statdb.dbx .
+model node (
```

Comment:
I think you should change `node` to be called `statdb_node` or something. `node` is too generic for how we are using this model right now.